.travis.yml

```diff
@@ -1,14 +1,14 @@
-language: node_js
-node_js:
-- "4"
-- "node"
 sudo: false
+language: node_js
 cache:
   directories:
-  - node_modules
-install:
-- npm install
-- npm update
+    - node_modules
+node_js:
+  - "4"
+  - "node"
+before_install:
+  - npm update
+  - npm prune
```

Comment:
Good idea using `npm prune`, since we're caching `node_modules/`.

I think we might see more confusing issues doing update in `before_install`, since then on install, if there are any issues with the build, we'll see them in the post install step of `npm install`. If we do the `npm update` after `npm install` we'll see those errors on the `npm build` step of the tests, which I think would be more intuitive.

I believe they fixed the behavior of running the `prepublish` script on `npm install`, so we can move that to `prepublishOnly` and then only build once per... build.